Incorporate cpprestsdk with AnyConnect Platform Libraries

seym45 opened the pull request:
- Enabled Position Independent Code (PIC) flag
- Added sample code to run on RPi
- Added build script for different platforms(andoird/RPi/ubuntu)

seym45 committed:
Incorporate cpprestsdk for AnyConnect
 Platform Libraries

 - Enabled Position independant code flag

seym45 committed:
Incorporate cpprestsdk for AnyConnect
Platform Libraries

- Added sample code to run on RPi

seym45 committed:
Incorporate cpprestsdk for AnyConnect
Platform Libraries

- Added build script for platforms

KhaledMahmood commented on build.ubuntu.sh:
```diff
@@ -0,0 +1,27 @@
+cd ${CPPREST_BUILD_DIR}
+pwd
+
+BOOST_ROOT=$HOME/projects/ac-access/.build_ubuntu/boost-1.61-ubuntu
```
Please use some standard path (e.g. $HOME/boost-1.61-ubuntu or <cpprest_root>/boost-1.61-ubuntu). ac-access is not related to this project.

seym45 replied:
Updated a patch here. https://github.com/anyconnect/cpprestsdk/pull/1/commits/97aa9e784f704d4bea5b1e5116026974e8ee3227

KhaledMahmood reviewed (approved):
Looks good.

seym45 committed:
Standardize Paths And Remove Irrelevant paths

- Changed paths with a generalized variable,
  need to pass as argument while running the script
- Directory structure is added for clarity

seym45 committed:
Fix `Build dir is not deleted'

- Fixed the bug
- And removed unnecessary submodule vcpkg
